iidm/iidm-xml-converter/src/main/java/com/powsybl/iidm/xml/SubstationXml.java

```diff
@@ -34,7 +35,13 @@ protected boolean hasSubElements(Substation s) {
 
     @Override
     protected void writeRootElementAttributes(Substation s, Network n, NetworkXmlWriterContext context) throws XMLStreamException {
-        context.getWriter().writeAttribute("country", context.getAnonymizer().anonymizeCountry(s.getCountry()).toString());
+        s.getCountry().ifPresent(country -> {
+            try {
+                context.getWriter().writeAttribute("country", context.getAnonymizer().anonymizeCountry(country).toString());
+            } catch (XMLStreamException e) {
+                throw new UncheckedXmlStreamException(e);
+            }
+        });
```

Comment:
This block seems simpler if you use a if statement

ˋˋˋ
if (s.getCountry().isPresent()) {
    context.getWriter().writeAttribute("country", context.getAnonymizer().anonymizeCountry(s.getCountry().get()).toString());
}
ˋˋˋ